tests/foreman/ui/test_syncplan.py

```diff
@@ -543,7 +547,7 @@ def test_positive_synchronize_custom_product_past_sync_date(self):
             # Verify product has not been synced yet
             sleep(delay/4)
             self.validate_repo_content(
-                product.name, repo.name,
+                product.name, repo,
```

Comment:
Now validate_repo_content() accepts product's name and repo as an entity. It's confusing mismatch. In any case, method's docstring should be updated. But i'd rather like to see:
1) (preferably) leave the helper as it was - accepting names. Then search for repo by its name + product name. It's a bit debatable as product name and repo name are unique only within the same org, so theoretically it may cause issues for us (although it's very unlikely to happen)
2) Update helper to accept product entity and repo entity
3) (worst case) Update helper's args to `product_name` and `repo`, so at least it will become intuitive.